bip-internalkey.mediawiki

```diff
@@ -0,0 +1,70 @@
+<pre>
+  BIP: TBD
+  Layer: Consensus (soft fork)
+  Title: INTERNALKEY
+  Author: Brandon Black <freedom@reardencode.com>
+  Comments-Summary: No comments yet.
+  Status: Draft
+  Type: Standards Track
+  Created: 2023-12-22
+  License: PD
+</pre>
+
+==Abstract==
+
+This BIP describes a new tapscript opcode (<code>OP_INTERNALKEY</code>) which
+pushes the taproot internal key to the stack.
+
+==Specification==
+
+When verifying taproot script spends having leaf version 0xc0 (as defined in
+BIP342), <code>OP_INTERNALKEY</code> replaces <code>OP_SUCCESS203</code>
+(0xcb). <code>OP_INTERNALKEY</code> pushes the taproot internal key, as defined
+in BIP341, to the stack.
+
+==Motivation==
+
+===Key spend with additional conditions===
+
+When building taproot outputs, especially those secured by an aggregate key
+representing more than 1 signer; the parties may wish to collaborate on signing
+with the taproot internal key, but only with additional script restrictions. In
```

Comment:
Can you give an example where using the key path with an aggregate key and having every participant enforce the additional restrictions themselves (i.e. only signing if they're met) wouldn't suffice?

Reply:
The most obvious case would be lightning symmetry where the parties want to pre-sign with CTV+CSFS to create a rebindable update transaction.